internal/ping.go

```diff
@@ -0,0 +1,43 @@
+package internal
+
+import (
+	"context"
+	"fmt"
+
+	"github.com/influxdata/influx-cli/v2/internal/api"
+)
+
+type Client interface {
```

Comment:
```suggestion
type PingClient interface {
```

Reply:
This pattern (defining a more focused interface instead of using the mega `api.ClientWithResponsesInterface`) is meant to help isolate unit test suites from one another. Adding a new route to the codegen ideally shouldn't require changes to unrelated tests.